radis/lbl/factory.py

```diff
@@ -1534,6 +1536,7 @@ def non_eq_spectrum(
         rot_distribution="boltzmann",
         overpopulation=None,
         name=None,
+        band_scaling=None,  # NEW: user-facing band scaling
```

Comment:
```suggestion
        band_scaling=None,  # user-facing band scaling
```
Won't be "new" in a few years